rasterio/rio/info.py

```diff
@@ -72,8 +72,12 @@ def env(ctx, key):
               help="Output extra information.")
 @click.option('--bidx', type=int, default=1,
               help="Input file band index (default: 1).")
+@click.option('--masked/--raw',
```

Comment:
@sgillies instead of ```--raw``` please consider ```--not-masked``` to more closely follow the true / false naming convention you've used for other flags elsewhere in ```rio```.    (I meant to respond to your earlier question about ```--raw``` but didn't get to it).